### PR TITLE
Add cachethq

### PR DESCRIPTION
**Pull request by daddz**

### CODE REVIEW

**Comment by rawmind0** [comment]
Hi @daddz ... Thanks for the PR and contribute....

I'm getting these error with your package....
```
2017-7-10 13:44:45Initializing Cachet container ...
2017-7-10 13:44:45Application key [base64:3Luehd4UTs9OFW6pu1y0GPo/Y3iwB/MACkHgW6IU7eY=] set successfully.
2017-7-10 13:44:45Please set base64:3Luehd4UTs9OFW6pu1y0GPo/Y3iwB/MACkHgW6IU7eY= as your APP_KEY variable in the environment or docker-compose.yml and re-launch

```

It seems that some env variable are missing...Looking at cachethq git repo,  https://github.com/CachetHQ/Docker/blob/master/docker-compose.yml
Installation docs https://docs.cachethq.io/v1.0/docs/installing-cachet

Could you please take a look?? Thanks...

**Comment by daddz** [comment]
@rawmind0: I added it as a variable

**Comment by rawmind0** [comment]
Thanks @daddz ... Package is working ok...

One question, don't you need to expose any port to access cachethq service?? If so, please add a lb to do it.

File: templates/cachethq/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      type: "password"
+      required: true
+      label: "PostgreSQL password"
+      default: "postgres"
+    
+    - variable: "VOLUME_DRIVER"
+      type: "string"
+      required: true
+      label: "Volume driver for PostgreSQL storage"
+      default: "local"
```
Comment (rawmind0) [inline templates/cachethq/0/rancher-compose.yml]: Selectable port to publish cachet with lb. 
Added health check.
``` 
+    - variable: "CACHET_PORT"
+      type: "int"
+      required: true
+      label: "Port to expose cachet"
+      default: 8000
+
+services:
+  lb:
+    scale: 1
+    start_on_create: true
+    lb_config:
+      certs: []
+      port_rules:
+      - priority: 1
+        protocol: http
+        service: cachet
+        source_port: ${CACHET_PORT}
+        target_port: 8000
+    health_check:
+      healthy_threshold: 2
+      response_timeout: 2000
+      port: 42
+      unhealthy_threshold: 3
+      initializing_timeout: 60000
+      interval: 2000
+      reinitializing_timeout: 60000
+  cachet:
+    scale: 1
+    start_on_create: true
+    health_check:
+      port: 8000
+      interval: 5000
+      unhealthy_threshold: 3
+      initializing_timeout: 60000
+      request_line: 'GET / HTTP/1.0'
+      healthy_threshold: 2
+      response_timeout: 5000 
+      reinitializing_timeout: 60000
```

File: templates/cachethq/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.catalog:
```
Comment (rawmind0) [inline templates/cachethq/0/rancher-compose.yml]: Version 2 format
```
-.catalog:
+version: '2'
+catalog:
```

**Review by rawmind0 — CHANGES_REQUESTED** [review]

**Comment by cjellick** [comment]
Very stale. Closing. Thanks! Please open a new PR if there is still a relevant change.

**Pull request closed without merging**